docs/AnaplanSink-sparksink.md

```diff
@@ -0,0 +1,24 @@
+# Anaplan sink
+
+ Description
+ -----------
+ Sink plugin for supporting data import from CDF to Anaplan server.
+
+ The placeholder with the same name as in the `Server file` filed must be predefined in the given Anaplan model.
+
+ Properties
```

Comment:
It would be good to have a section on supported data types. See https://github.com/data-integrations/google-cloud/blob/develop/docs/BigQueryTable-batchsink.md#data-type-mappings-from-cdap-to-bigquery for an example.

Reply:
Thanks, updated!